Lib/enum.py

```diff
@@ -719,7 +719,7 @@ def __contains__(self, other):
         if not isinstance(other, self.__class__):
             raise TypeError(
                 "Unsupported operands type(s) for 'in': '%s' and '%s'" % (
-                    type(other).__qualname__, type(self).__qualname__))
+                    type(other).__qualname__, self.__qualname__))
```

Comment:
`self` is the member itself, not the class -- that is why the `__qualname__` lookup is faling; use `self.__class__._qualname__` instead.